src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -423,6 +429,16 @@ contract SecurityCouncilNomineeElectionGovernor is
         return _elections[proposalId].isContender[possibleContender];
     }
 
+    /// @notice Recover EIP712 signature for `AddContenderMessage`
+    function recoverAddContenderMessage(uint256 proposalId, address contender, bytes calldata signature) public view returns (address) {
+        bytes32 digest = _hashTypedDataV4(keccak256(abi.encode(
+            keccak256("AddContenderMessage(uint256 proposalId,address contender)"),
```

Comment:
The contender arg is a bit redundant

Reply:
e6c035eebb6636bf45a037e543da6ff8a7984e44